src/MC4R/ParseMC4R.py

```diff
@@ -38,3 +38,38 @@ def parse_year_of_birth(year_of_birth: int) -> str:
         f'but was {year_of_birth}')
     return f'{year_of_birth}-01-01T00:00:00Z'
     
+
+
+
+def parse_date_of_diagnosis(year,month,day):
```

Comment:
Please add data types for year, month and day as e.g. `year: int`. Seems to be int, but please double check. Also missing spaces before `,`

Reply:
done, it was strings and passes them over to an int now